src/main/java/TaskList.java

```diff
@@ -0,0 +1,158 @@
+import java.util.Scanner;
+public class TaskList {
+        private final Task[] taskList;
+        private static final int MAX_TASKS = 100;
+        //protected boolean[] isMarked;
+        private int taskCount;
+
+
+        public TaskList() {
+            this.taskList = new Task[MAX_TASKS];
+            this.taskCount = 0;
+        }
+
+    private void processTaskCommand(String command) {
```

Comment:
You could consider refactoring parts of this method to separate methods to avoid long methods and adhere to this code quality [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-deep-nesting)